server/server/search/dictionaries.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+import lxml.html
+
+from elasticsearch import ConnectionError
+from flask import request, current_app
+from flask_restful import Resource
+
+from search import query as query_search
+from collections import Counter
+
+
+from common.arangodb import get_db
+
+def cleanly_truncate_html(html_string, length):
+    # TODO: The function name is aspirational, at the moment it crudely
+    # butchers the HTML. Need to discuss further before choosing implementation.
+    content = html_string
+    if len(content) >= length:
+        return content[:length] + '…'
+    else:
+        return content
+    
+def search(word, truncate_length=1000):
+    db = get_db()
+    
+    results = list(db.aql.execute('''
+        FOR doc IN dictionary_full
+            FILTER doc.word == @word OR doc.word_ascii == @word
+            RETURN KEEP(doc, "dictname", "lang_to", "lang_from", "word", "word_ascii", "text")
+    ''',
+    bind_vars={'word': word}))
```

Comment:
continuation line under-indented for visual indent
https://www.python.org/dev/peps/pep-0008/#indentation